src/operator/elemwise_op_common.h

```diff
@@ -73,6 +73,12 @@ inline bool ElemwiseStorageAttr(const nnvm::NodeAttrs& attrs,
     dispatched = storage_type_assign(out_attrs, kCSRStorage,
                                      dispatch_mode, dispatch_ex);
   }
+  if (!dispatched && (((*in_attrs)[0] == kDefaultStorage && (*in_attrs)[1] == kCSRStorage) ||
+                      ((*in_attrs)[0] == kCSRStorage && (*in_attrs)[1] == kDefaultStorage))) {
+    // dense, csr -> csr
+    dispatched = storage_type_assign(out_attrs, kDefaultStorage,
```

Comment:
1. Changing `ElemwiseStorageAttr` will affect ALL operators using this function. 
2. Unary op also uses this function. There's no guarantee len(in_attr) > 1.

Reply:
What about I add a check for # of in_attrs and out_attrs?